cypress/common/i-have-a-legacy-beacon.spec.ts

```diff
@@ -1,22 +1,25 @@
-import axios from "axios";
 import { ILegacyBeaconRequest } from "../../src/gateways/interfaces/LegacyBeaconRequest";
 
 export const iHavePreviouslyRegisteredALegacyBeacon = async (
   legacyBeaconRequest: ILegacyBeaconRequest
 ): Promise<void> => {
   const url = `${Cypress.env("API_URL")}/migrate/legacy-beacon`;
-  cy.request("/api/auth/session").then(async () => {
```

Comment:
Request to "/api/auth/session" wasn't necessary here, as the migrations endpoint is only behind BasicAuth